migrate/repo_checks.py

```diff
@@ -0,0 +1,364 @@
+"""
+Run checks Against Repos and correct them if they're missing something.
+
+"""
+from itertools import chain
+from pprint import pformat
+
+import click
+import requests
+from fastcore.net import HTTP4xxClientError, HTTP5xxServerError, HTTP404NotFoundError
+from ghapi.all import GhApi, paged
+
+
+class Check:
+    def __init__(self, api, org, repo):
+        raise NotImplementedError
```

Comment:
Why specify this `__init__` and not implement anything? You could give a default that assigns these attributes to member variables and then have subclasses implement their own customized ones and call this via `super().__init__()`